go.mod

```diff
@@ -3,14 +3,13 @@ module github.com/Azure/azure-service-bus-go
 go 1.12
 
 require (
-	github.com/Azure/azure-amqp-common-go/v3 v3.0.1
-	github.com/Azure/azure-sdk-for-go v37.1.0+incompatible
-	github.com/Azure/go-amqp v0.13.1
-	github.com/Azure/go-autorest/autorest v0.11.7
-	github.com/Azure/go-autorest/autorest/adal v0.9.4
+	github.com/Azure/azure-amqp-common-go/v3 v3.1.0
+	github.com/Azure/azure-sdk-for-go v51.1.0+incompatible
+	github.com/Azure/go-amqp v0.13.3
+	github.com/Azure/go-autorest/autorest v0.11.18
```

Comment:
should I keep these version bumps for another PR?

Reply:
The go-autorest and azure-sdk-for-go should be fine as they're used for tests.

Updating common might be useful as it adds support for managed service identity which people have been asking for.  I don't think simply makes that work but it's a step in the right direction.

Reply:
I have the msi implementation in go-shuttle, so we can contribute it back here